README.rst

```diff
@@ -1,90 +1,113 @@
+4. Andrea Grisafi, Alan M. Lewis, Mariana Rossi, Michele Ceriotti, "Electronic-Structure Properties from Atom-Centered Predictions of the Electron Density", *Journal of Chemical Theory and Computation* **19**, 4451 (2023) [https://pubs.acs.org/doi/10.1021/acs.jctc.2c00850]
 
 Installation
 ------------
 In the SALTED directory, simply run :code:`make`, followed by :code:`pip install .`
    
 Dependencies
 ------------
-- **rascaline**: rascaline installation requires a RUST compiler. To install a RUST compiler, run:
+
+--> **rascaline**: rascaline installation requires a RUST compiler. To install a RUST compiler, run:
 :code:`curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh && source "$HOME/.cargo/env"`
 rascaline can then be installed using
 :code:`pip install git+https://github.com/Luthaf/rascaline.git`
 
-- **mpi4py**: mpi4py is required to use MPI parallelisation; SALTED can nonetheless be run without this.
+--> **mpi4py**: mpi4py is required to use MPI parallelisation; SALTED can nonetheless be run without this.
 A parallel h5py installation is required to use MPI parellelisation. This can be installed by running:
 :code:`HDF5_MPI="ON" CC=mpicc pip install --no-cache-dir --no-binary=h5py h5py`
 provided HDF5 has been compiled with MPI support.
 
-Training dataset
-----------------
-Training data consists in the expansion coefficients of the scalar field over atom-centered basis functions made of radial functions and spherical harmonics. We assume to work with real spherical harmonics defined with the Condon-Shortley phase convention. No restriction is instead imposed on the nature of the radial functions. Because of the non-orthogonality of the basis, the overlap matrix between the basis functions is also required as input. The size of these arrays has to correspond to the number of atoms, as sorted in the geometry file, times the number of basis functions belonging to each atom. The ordering of the basis set must follow the structure: 
+Input file
+----------
+SALTED input is provided in a :code:`inp.yaml` file, which is structured in the following sections:
+
+- :code:`salted` (required): define root storage directory and workflow label 
+
+- :code:`system` (required): define system parameters 
+
+- :code:`qm` (required): define information about quantum-mechanical reference
+
+- :code:`descriptor` (required): define parameters of symmetry-adapted descriptors
 
-- For a given atomic species X, cycle over angular momenta L 
+- :code:`gpr` (required): define Gaussian Process Regression parameters 
 
-- For each (X,L), cycle over radial functions n 
+- :code:`prediction` (optional): manage predictions on unseen datasets  
 
-- For each (X,L,n), cycle over the angular functions sorted as -L,...,0,...,+L
+Input Dataset
+-------------
+Input structures are required in XYZ format; the corresponding filename must be specified in the :code:`inp.system.filename`. 
```

Comment:
I think you really mean extxyz. At least that is what we were using overall, because of the lattice parameters, etc. This should be amended, especially important for periodic systems. Can link ASE description here.